README.md

```diff
@@ -244,71 +271,75 @@ AwsSsmEnv.load!(
 
 #### other
 
-`AwsSsmEnv::NamingStrategy`を実装したクラスのインスタンス、もしくは`parse_name`メソッドを持つ  
-インスタンスを指定した場合はそのインスタンスをそのまま利用する。
+If you specify an instance of a class that implements `AwsSsmEnv::NamingStrategy` in` fetch`,
+or an instance with a `parse_name` method, use that instance as is.
+
 
 ### path: [String]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合は必須となる。  
-パラメータを取得するパス階層を指定する。  
-下の例では`/myapp/web/production`直下のパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
```

Comment:
`It` is a bit ambiguous, so you should write like this:

> This option is required if nothing is specified for ...

Or, you can ommit the subject like this:

> Required if nothing is specified for ...